### DISCUSSION TITLE
Add ability to like, i. e. add images to favorites

### DISCUSSION THREAD

**Discussion by user622628252416** [Radiokot/photoprism-android-client#99] [Ideas]
I like using favorites to organize my images while viewing them, so being able to do that in the app would be really helpful to me.

At first glance, that feature seems fairly straightforward to implement. As far as I can tell, adding/removing images from favorites is just a POST/DELETE api call to the /api/v1/photos/{uid}/like endpoint.

Expanding on that idea, archiving images from inside the app would also be pretty useful, but not quite as important to me.

Do you agree such a feature would be a sensible addition? I haven't really looked into the code yet, but if you want help, I might contribute.

**Comment by Radiokot** [comment]
Hi. I agree toggling the like from the app is useful, it is in the roadmap. I think this feature can be added to one of the upcoming updates.

**Comment by Radiokot** [comment]
Hi there. Please, check out version [1.21.0](https://github.com/Radiokot/photoprism-android-client/releases/tag/1.21.0(34)) – it has the ability to like/dislike content as well as see only liked (favorite) photos.
